data/mods/ssb/moves.js

```diff
@@ -231,6 +231,35 @@ let BattleMovedex = {
 		target: "normal",
 		type: "Normal",
 	},
+	// Akiamara
+	x1: {
+		accuracy: 100,
+		basePower: 100,
+		category: "Special",
+		desc: "This move becomes a physical attack if the user's Attack is greater than its Special Attack, including stat stage changes. This move and its effects ignore the Abilities and stat changes of other Pokemon.",
+		shortDesc: "Physical if user's Atk > Sp. Atk. Ignores Abilities & stat changes.",
+		id: "x1",
+		isNonstandard: "Custom",
+		name: "x1",
+		pp: 10,
+		priority: 0,
+		flags: {protect: 1, mirror: 1},
+		onTryMove() {
+			this.attrLastMove('[still]');
+		},
+		onModifyMove(move, pokemon) {
+			if (pokemon.getStat('atk', false, true) > pokemon.getStat('spa', false, true)) move.category = 'Physical';
+		},
+		onPrepareHit(target, source) {
+			this.add('-anim', source, 'Focus Punch', target);
+		},
+		ignoreAbility: true,
+		ignoreEvasion: true,
+		ignoreDefensive: true,
```

Comment:
The custom dosent do any of these?

Reply:
It does. "Effects: Physical if user's Atk > Sp. Atk. Ignores Abilities and stat stage changes."